internal/provider/schema_cloudflare_account.go

```diff
@@ -0,0 +1,28 @@
+package provider
```

Comment:
you'll want to add the documentation examples and then run `make docs` to have the schema generate the documentation for you.

Reply:
examples added